src/buildkite_test_collector/collector/run_env.py

```diff
@@ -115,7 +115,9 @@ def as_json(self) -> Dict[str, str]:
             "branch": self.branch,
             "commit_sha": self.commit_sha,
             "message": self.message,
-            "url": self.url
+            "url": self.url,
+            "collector": 'python-{COLLECTOR_NAME}',
```

Comment:
I don't _think_ I should be changing gem/package/library name in the setup.py, so I have instead added collector type clarity here by interpolating the language before the generic name (buildkite-test-collector is the same as the exlir and ruby collector names. The JS collector follows this pattern of interpolation the package name with the language).